### PR TITLE
Doc changes for 1.13.35.1-beta

### PR DESCRIPTION
**Pull request by oschaaf**
Initial draft for 1.13.35.1-beta -- not ready yet.
Once everyone is happy and this is merged, this should be backported to branch 35

### CODE REVIEW

File: html/doc/release_notes.html [hunk 1]
```diff
@@ -9,7 +9,123 @@
...
+  <h3 class="hide-from-toc">New Features</h3>
+  <dl>
+    <!-- @@TODO@@: Link future CSP doc -- or remove the anchor. -->
+    <dt><a href="system#csp">Content Security Policy Support</a></dt>
+    <dd>
+      Opt-in feature enabling optimizations to adapt to Content Security Policies.
+    </dd>
+    <dt>Allow UrlValuedAttribute:image on elements with children</dt>
+    <dd>
+       Tag that have children are now considered acceptable targets for image optimization.
```
Comment (jmarantz) [inline html/doc/release_notes.html]: s/Tag/Tags/

File: html/doc/release_notes.html [hunk 2]
```diff
@@ -9,7 +9,123 @@
...
+    </dd>
+    <dt><a href="system#redis">Support specifying a Redis Database</a></dt>
+    <dd>
+      It is now possible to specify which database will be used when configuring Redis as a
+      cache.
+    </dd>
+    <dt>Sync with WebP change in Chromium</dt>
+    <dd>
+        @@TODO@@
```
Comment (jmarantz) [inline html/doc/release_notes.html]: Background:

Mozilla WebP Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1294490
Chrome WebP Bug: https://bugs.chromium.org/p/chromium/issues/detail?id=649264
WebM discussion: https://groups.google.com/a/webmproject.org/forum/#!msg/webp-discuss/rJ2a0j1BNQI/XXqK918ZEQAJ

File: html/doc/release_notes.html [hunk 3]
```diff
@@ -9,7 +9,123 @@
...
+      When over 50% of http fetch attempts inside a 30-minute period fail, pagespeed will now emit
+      an error entry to the logs.
+    </dd>
+    <dt>Standby Mode</dt>
+    <dd>
+        In standby mode PageSpeed is off, except it serves .pagespeed. resources and
+        PageSpeed query parameters are interpreted. This is equivalent to "off" in
+        mod_pagespeed. Previously, ngx_pagespeed had no equivalent.
+        
+        With this change, "off" in mod_pagespeed is deprecated, and people should use
```
Comment (jmarantz) [inline html/doc/release_notes.html]: s/people should use "standby"/"standby" should be used/

File: html/doc/release_notes.html [hunk 4]
```diff
@@ -9,7 +9,123 @@
...
+  <h4 class="hide-from-toc">Issues Affecting only Nginx</h4>
+  <ul>
+    <li><strong>
+        <a href="https://github.com/pagespeed/ngx_pagespeed/issues/1380">
+         #1380</a></strong>
+         Nginx worker 100% cpu usage (spinning on write returning EAGAIN).</li>
+    <li><strong>
+        <a href="https://github.com/pagespeed/ngx_pagespeed/issues/1375">
+         #1375</a></strong>
+         Don't respond with an entitybody to HEAD requests for html.</li>
```
Comment (jmarantz) [inline html/doc/release_notes.html]: should this be "entity body"?  Or is there supposed to be no space there?

**Comment by oschaaf** [comment]
@jmarantz updated to address your comments. Does the WebP change description look good?

**Comment by oschaaf** [comment]
I think I am ready with this. I also included some other doc changes in here so we can discuss that in one go. Once this is merged, I think there's one final doc change that will have to go in for te release: the date that it will get shipped.

**Review by jmarantz — APPROVED** [review]

**Pull request merged**